ui/lang/sv-SE.json

```diff
@@ -0,0 +1,44 @@
+{
+  "/AmWqb": "+ Omslagsbild",
+  "0KKXrH": "Budget",
+  "26gDq9": "Finansieringsmål:",
+  "3g96gM": "Inga bidrag än",
+  "47FYwb": "Cancel",
+  "4U2uvI": "Completed on {day}",
+  "7dyN5S": "Accept funding",
+  "C/OtcW": "Non-monetary contributions",
+  "Ctmnwu": "Fund",
+  "DYsdAN": "Can't find this bucket...",
+  "FHJLAL": "+ Sammanfattning",
+  "HSBYFr": "allowed.",
+  "J2D0+i": "Mark as completed",
+  "Jltspt": "+ Bilder",
+  "K3r6DQ": "Delete",
+  "Lvy00l": "Existing funds",
+  "MJ2jZQ": "Total",
+  "NYQP6J": "stretch goal:",
+  "NaGSue": "Max.",
+  "OXKkW2": "Yes, cancel it",
+  "Q6iAKB": "Are you sure you want to cancel this {bucketName}?",
```

Comment:
When we translate this and similar cases, I wonder how we should handle the names we get from env vars. One env var per word and language can't work. Maybe the most reasonable solution is to only allow custom nouns when the user is on english? @aerugo

Reply:
fixed